### ISSUE TITLE
Editor role cannot update most content even when it has permission to update

### ISSUE THREAD

**Issue by amy1hwang** [directus/directus#23273]
### Describe the Bug

The admin role can still update all contents but the editor cannot in not all, but most contents even though the permission is there. This seemed to start happening starting yesterday. When I gave the role admin access, the role was able to edit. When I revoke the admin access and tried revoking permission to create, read, delete and then giving back full access to create, read, and delete, it still did not have the ability to update contents.

The screenshots are examples.
<img width="1225" alt="Screenshot 2024-08-09 at 10 59 18 AM" src="https://github.com/user-attachments/assets/f3d98428-eb1c-444c-a578-f7c27253084b">
<img width="923" alt="Screenshot 2024-08-09 at 11 01 15 AM" src="https://github.com/user-attachments/assets/dd379d75-c82a-43a5-a075-e07a172455ee">

### To Reproduce

1. Make sure a non-admin role has permission to at least read and update to a collection.
2. Check the collection to see if it's possible to edit.

### Directus Version

v11.0.1

### Hosting Strategy

Directus Cloud

**Comment by hanneskuettner** [comment]
Is that a singleton collection?

**Comment by hanneskuettner** [comment]
If it is a singleton collection, this is the same issue as https://github.com/directus/directus/issues/23268 it has been addressed already and will be resolved in the upcoming release.

**Comment by amy1hwang** [comment]
Yes, now that you asked, I've checked all the collections and it looks like only the singletons are having this issue. Thank you for your quick reply and looking forward to the next release.
